### PR TITLE
Add lit_encounter, checked against cards

### PR DESCRIPTION
**Pull request by drawntotheflame**
Adding A Light in the Fog encounter cards.

### CODE REVIEW

**Comment by drawntotheflame** [comment]
@zzorba do you have a quiet minute to check these?

**Comment by zzorba** [comment]
Yep, I can review this later today.

File: pack/tic/lif_encounter.json [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+        "back_flavor": "A thick padlock bars your way into what you suspect is a small cellar below the lighthouse.",
+        "back_illustrator": "Yoann Boissonnet",
+        "back_name": "Lighthouse Basement",
+        "back_text": "The door leading into the basement of the lighthouse is locked tight. You cannot enter this location unless you control the blue key.",
+        "clues": 0,
+        "code": "07244",
+        "double_sided": true,
+        "encounter_code": "a_light_in_the_fog",
+        "encounter_position": 14,
+        "faction_code": "mythos",
```
Comment (zzorba) [inline pack/tic/lif_encounter.json:265]: Missing `"flavor": "Well, that's certainly not a cellar",`

File: pack/tic/lif_encounter.json [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+        "name": "Captured!",
+        "pack_code": "lif",
+        "position": 252,
+        "quantity": 1,
+        "text": "<b>The first time an investigator is captured, read the following:</b>\n<i>You are bound and dragged deep underground, then thrown into a sealed prison cell, surrounded on all sides by dripping stone walls. Your cries fall on deaf ears as your captor's footsteps fade into the distance.</i>\nRemove the captured investigator from their location and return each asset in that investigator's hand slots to their hand.\nTurn that investigator's mini-card sideways to indicate that they have been captured. Captured investigators cannot move, fight, or be engaged.\nFlip this story card over to its \"Holding Cells\" side and put it into play. It is not on any existing row or column, and is only connected to Sunken Grotto locations through its connection icons. Place the captured investigator in Holding Cells.\nFrom now on, whenever an investigator is captured, resolve this same effect (return each asset in their hand slots to their hand, turn their mini-card sideways, and place them in Holding Cells).",
```
Comment (zzorba) [inline pack/tic/lif_encounter.json:427]: I believe the flavor here should have the `<blockquote>` treatment around the `<i>` tag?

**Review by zzorba — APPROVED** [review]
Two small corrections, otherwise looks good to go

**Comment by drawntotheflame** [comment]
Thank you @zzorba 

@Kamalisk this can go live when you're ready.

**Pull request merged**